deploy/examples/export-external-cluster.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+###############################
+
+USAGE=$(cat <<-'EOF'
+# download the create-external-cluster-resource.py script
+pushd /tmp || exit
+curl -LOs https://raw.githubusercontent.com/rook/rook/release-1.10/deploy/examples/create-external-cluster-resources.py
+popd || exit
+
+# cp script into toolbox pod
+toolbox=$(kubectl -n rook-ceph get pod -l app=rook-ceph-tools -o jsonpath='{.items[*].metadata.name}')
+kubectl -n rook-ceph cp /tmp/create-external-cluster-resources.py $toolbox:/etc/ceph/
+
+# run script via the toolbox pod (you have to specify what you are exporting here, e.g. 'rbd-data-pool-name')
```

Comment:
Can you please explain the use case, 

I see you are just replacing `apply` to `create` which we can directly do in the import script.

Or there is something else gitpos would be needing?